catalog-rest-service/src/main/java/org/openmetadata/catalog/events/ChangeEventHandler.java

```diff
@@ -79,7 +79,17 @@ public Void process(ContainerRequestContext requestContext, ContainerResponseCon
         for (var thread : CommonUtil.listOrEmpty(getThreads(responseContext))) {
```

Comment:
can we make this as part of import it self import CommonUtil.listOrEmpty

Reply:
Sure, let me fix that